### PR TITLE
Update  Roslyn codeflow for 16.5 P1 snap

### PR DESCRIPTION
**Pull request by JoeRobich**

### CODE REVIEW

File: src/GitHubCreateMergePRs/config.xml [hunk 1]
```diff
@@ -6,19 +6,19 @@
...
     <merge from="release/dev16.1" to="release/dev16.2" />
     <merge from="release/dev16.2" to="release/dev16.3" />
-    <merge from="release/dev16.3" to="release/dev16.4-preview2" />
-    <merge from="release/dev16.4-preview2" to="release/dev16.4" />
-    <merge from="release/dev16.4" to="master" />
-
+    <merge from="release/dev16.3" to="release/dev16.4" />
+    <merge from="release/dev16.4" "release/dev16.5-preview1" />
+    <merge from="release/dev16.5-preview1" to="master" />
+
```
Comment (RikkiGibson) [inline src/GitHubCreateMergePRs/config.xml]: trailing whitespace.

Comment (JoeRobich) [reply src/GitHubCreateMergePRs/config.xml]: stupid github editor

File: src/GitHubCreateMergePRs/config.xml [hunk 2]
```diff
@@ -6,19 +6,19 @@
...
     <!-- Roslyn branches (between vs-deps branches) -->
     <merge from="dev15.0.x-vs-deps" to="dev15.9.x-vs-deps" />
     <merge from="dev15.9.x-vs-deps" to="dev16.0-vs-deps" />
     <merge from="dev16.0-vs-deps" to="release/dev16.1-vs-deps" />
     <merge from="release/dev16.1-vs-deps" to="release/dev16.2" />
     <merge from="release/dev16.2" to="release/dev16.3-vs-deps" />
-    <merge from="release/dev16.3-vs-deps" to="release/dev16.4-preview2-vs-deps" />
-    <merge from="release/dev16.4-preview2-vs-deps" to="release/dev16.4-vs-deps" />
-    <merge from="release/dev16.4-vs-deps" to="master-vs-deps" />
+    <merge from="release/dev16.3-vs-deps" to="release/dev16.4-vs-deps" />
```
Comment (RikkiGibson) [inline src/GitHubCreateMergePRs/config.xml]: these look wrong (most of these are vs-deps to vs-deps on the same release train, but others are not)

File: src/GitHubCreateMergePRs/config.xml (hunk 1 above)
Comment (JoeRobich) [reply src/GitHubCreateMergePRs/config.xml]: I want my monaco editor

File: src/GitHubCreateMergePRs/config.xml (hunk 2 above)
Comment (JoeRobich) [reply src/GitHubCreateMergePRs/config.xml]: This comment should be earlier in teh file `<!-- Roslyn does not have a dev16.2-vs-deps branch as it was merged back into dev16.2 -->`

Comment (JoeRobich) [reply src/GitHubCreateMergePRs/config.xml]: Fixed by removing 16.1 and 16.2 from code flow

**Pull request merged**